src/client/app/teams/select-teams.component.html

```diff
@@ -23,10 +23,10 @@
 	<div *ngIf="showTags" class="col-md-6">
 		<div style="padding-left:0;" class="col-md-12 pull-left btn-group" dropdown>
 			<button style="text-align: left;" type="button" class="form-control input-sm dropdown-toggle hide-overflow" dropdownToggle>
```

Comment:
This is not a question that should hold up this fix, but is there a reason we aren't using a combobox here?